packages/app-council/src/Motions/Propose.tsx

```diff
@@ -14,17 +14,17 @@ import { withApi, withCalls, withMulti } from '@polkadot/ui-api';
 
 import translate from '../translate';
 
-type Props = TxModalProps & ApiProps & {
+interface Props extends TxModalProps, ApiProps {
```

Comment:
Nice! Ok, we should probably apply this everywhere and only have interfaces (unless it is a simple `type Props = BareProps`)

Not for here, but something to keep in mind as we touch stuff.